jira-create-issue/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "jira-create-issue",
+  "version": "3.0.0",
+  "description": "A GitHub Action to create a Jira issue.",
+  "main": "main.js",
+  "scripts": {
+    "prepare": "ncc build -m main.js -o dist/main --license licenses.txt",
+    "lint": "eslint ."
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/metro-digital/cf-github-actions.git"
+  },
+  "keywords": [
+    "GitHub",
+    "Actions",
+    "GKE",
+    "IP",
+    "Allowlist",
+    "Control",
+    "Plane"
+  ],
```

Comment:
It seems like that you forgot to update the keywords after copying the template from the GKE control plane authorised networks updater action.

Reply:
Yeah, I was trying to figure out the whole npm run prepare stuff and forgot about this, sry.